netbout-client/src/test/java/com/netbout/client/RtAttachmentsITCase.java

```diff
@@ -85,4 +86,50 @@ public void postsAndReads() throws Exception {
         attachments.delete(name);
     }
 
+    /**
+     * RtAttachments can obtain author of an attachment.
+     * @throws Exception If there is some problem inside
+     */
+    @Test
+    public void obtainsAuthor() throws Exception {
+        final User user = NbRule.get();
+        final Alias alias = user.aliases().iterate().iterator().next();
+        final Inbox inbox = alias.inbox();
+        final Bout bout = inbox.bout(inbox.start());
+        final Attachments attachments = bout.attachments();
+        final String name = "name";
+        attachments.create(name);
+        final Attachment attachment = attachments.get(name);
+        MatcherAssert.assertThat(
+            attachment.author(), Matchers.equalTo(alias.name())
+        );
+        attachments.delete(name);
+    }
+
+    /**
+     * RtAttachments can obtain the creation date of an attachment.
+     * @throws Exception If there is some problem inside
+     */
+    @Ignore
+    @Test
+    public void obtainsCreationDate() throws Exception {
+        final User user = NbRule.get();
+        final Alias alias = user.aliases().iterate().iterator().next();
+        final Inbox inbox = alias.inbox();
+        final Bout bout = inbox.bout(inbox.start());
+        final Attachments attachments = bout.attachments();
+        final long before = System.currentTimeMillis();
+        final String name = "attach-name";
+        attachments.create(name);
+        final Attachment attachment = attachments.get(name);
+        MatcherAssert.assertThat(
+            attachment.date().getTime(), Matchers.greaterThan(before)
+        );
+        final long after = System.currentTimeMillis();
+        MatcherAssert.assertThat(
+            attachment.date().getTime(), Matchers.lessThan(after)
+        );
+        attachments.delete(name);
```

Comment:
@mkordas I have seen it in many of the Attachment related tests, so I thought it was convenient to have it.